src/annotator/bucket-bar-client.js

```diff
@@ -0,0 +1,58 @@
+import { ListenerCollection } from '../shared/listener-collection';
+
+/**
+ * @typedef {import('../shared/bridge').Bridge<GuestToHostEvent, HostToGuestEvent>} HostBridge
+ * @typedef {import('../types/annotator').Destroyable} Destroyable
+ * @typedef {import('../types/bridge-events').HostToGuestEvent} HostToGuestEvent
+ * @typedef {import('../types/bridge-events').GuestToHostEvent} GuestToHostEvent
+ */
+
+/**
+ * Communicate to the `host` frame when:
+ * 1. The set of anchors has been changed (due to annotations being added or removed)
+ * 2. The position of anchors relative to the viewport of the guest has changed
+ *
+ * @implements Destroyable
```

Comment:
I notice we mix `@implements Name` and `@implements {Name}` in our code. It seems that TS accepts either, but we should standardize. I'm inclined to include the parens as this seems to be the preferred style from the [TS handbook](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#implements).